src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplate.utils.ts

```diff
@@ -0,0 +1,77 @@
+import { v4 } from 'uuid';
+
+import i18n from 'i18n';
+import { Item, PhrasalTemplateField, PhrasalTemplateFieldType } from 'redux/modules';
+import { ItemResponseType } from 'shared/consts';
+
+// TODO: M2-7211 — Additional response types to be added with
+// additional UI for selecting discrete response values.
+const PHRASAL_TEMPLATE_COMPATIBLE_RESPONSE_TYPES = [
+  ItemResponseType.Date,
+  ItemResponseType.MultipleSelection,
+  ItemResponseType.NumberSelection,
+  ItemResponseType.SingleSelection,
+  ItemResponseType.Slider,
+  ItemResponseType.Text,
+  ItemResponseType.Time,
+  ItemResponseType.TimeRange,
+  // ItemResponseType.ABTrails,
+  // ItemResponseType.Flanker,
+  // ItemResponseType.MultipleSelectionPerRow,
+  // ItemResponseType.SingleSelectionPerRow,
+  // ItemResponseType.SliderRows,
+  // ItemResponseType.StabilityTracker,
+  // ItemResponseType.TouchPractice,
+  // ItemResponseType.TouchTest,
```

Comment:
It looks like you've partially implemented [M2-7211](https://mindlogger.atlassian.net/browse/M2-7211) in this PR – at least the "Display Mode" dropdown is included. But not the "Response From" dropdown yet.

Since "Display Mode" is already supported, I don't see why we need to keep having `MultipleSelectionPerRow` and `SingleSelectionPerRow` commented out here, as I tested them and they both were working fine:

![CleanShot 2024-08-08 at 16 33 37@2x](https://github.com/user-attachments/assets/1064366c-407a-43cd-bc85-8465c3287351)

```suggestion
  ItemResponseType.MultipleSelectionPerRow,
  ItemResponseType.SingleSelectionPerRow,
  // ItemResponseType.SliderRows,
```

And presumably you're still adding support for the "Response From" dropdown required by `SliderRows` in [M2-7211](https://mindlogger.atlassian.net/browse/M2-7211), which I see is still in progress.

As for the other commented item types here: `ABTrails`, `Flanker`, `StabilityTracker`, etc., I don't believe response values for any of those types are supported by Phrasal Template, and I think you can probably just delete those commented lines.

Reply:
I'm guessing this was merged in #1881?
As mentioned on that PR, please try and keep these tickets separate. This is already a large PR due to the fact that M2-7076 is an 8-pt ticket. 

If there's any remaining work needed for finishing off [M2-7211](https://mindlogger.atlassian.net/browse/M2-7211), then I suggest leaving that to another PR for that ticket. We need to make this ticket and it's scope in M2-7076 top priority and merged first.